main.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/gutakk/go-google-scraper/controllers"
 	"github.com/gutakk/go-google-scraper/db"
 	"github.com/gutakk/go-google-scraper/migration"
+	"github.com/gutakk/go-google-scraper/oauth"
+)
+
+const (
+	startServerFail = "Failed to start the server %v"
```

Comment:
Overall, be more specific in the naming of constants as I am often a bit lost of the usage of each constant. Some are to define view name while other error messages. So how about using a suffix to be more clear:

```suggestion
	startServerFailureError = "Failed to start the server %v"
```
or

```suggestion
	startServerFailureMessage = "Failed to start the server %v"
```

Reply:
Good point. Fixed in 8b3ce7a0c3a1f4848846110992dce4d2f1748a9b